executor/src/witgen/identity_processor.rs

```diff
@@ -111,15 +153,18 @@ impl<'a, 'b, T: FieldElement> IdentityProcessor<'a, 'b, T> {
             return result;
         }
 
-        for m in self.machines.iter_mut() {
-            // TODO also consider the reasons above.
-            if let Some(result) = m.process_plookup(
-                self.fixed_data,
-                self.fixed_lookup,
-                identity.kind,
-                &left,
-                &identity.right,
-            ) {
+        for i in 0..self.machines.len() {
+            if let Some(result) = self.machines.split(i, |current, others| {
+                // TODO also consider the reasons above.
```

Comment:
@chriseth Do you actually know what this comment means? I think it's been copied around quite a bit recently, but I'm not sure if it's still relevant.

Reply:
At some point, we were ignoring failures from some identities. I'm not sure if we are still collecting reasons for failure. We were just ignoring some reasons and did not put them into the vector where we collected the failures.